GeorgianMemenator/Assets.xcassets/Icons/9gag.imageset/Contents.json

```diff
@@ -1,7 +1,6 @@
 {
   "images" : [
     {
-      "filename" : "9gag.svg",
```

Comment:
You deleted only the image, but we need to delete entire file. So, please just delete entire fil

<img width="218" alt="image" src="https://user-images.githubusercontent.com/9494262/208642002-346d6471-7c40-4a38-a973-f54d7c522067.png">

Reply:
Should  I delete only 9gag file? Does it need some changes in terminal?

Reply:
I have deleted the file of 9gag, is it correct now? I haven't done anything else.

Reply:
I think to make a git push on terminal, but I am not quite sure, that's why I am waiting to you

Reply:
You need to delete file in Xcode and thats it. After that just push the changes. I think it should be correct, if not we can easily revert the changes, dont worry.

Reply:
I have done everything, please check